guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,31 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You download Ansible Galaxy from the AppStream RHEL 8 repository.
+If you need support regarding `/usr/bin/ansible-playbook`, you can reach out to Red Hat support, but we do not provide support for the content you try to execute with it.
+* You buy an Ansible subscription, configure `ansible-galaxy` to talk to `cloud.redhat.com` and use it to download supported collections from there.
+If you need support, you reach out to the Ansible team.
+* You rewrite your templates, roles, etc.
+
+To find out what content is affected after the upgrade, you can run:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ansible-lint -t only-builtins -p _path/to/your/playbook/play.yml_ ...
+----
```

Comment:
this rule (only-builtins) was added in ansible-lint 6.1.0 (and then there was an annoying false-postive bug, which only got fixed in 6.3.0), which is not packaged at all yet for any platform we support
so while yes, this is a good way to check it, we cannot recommend it to users, as they have no way of obtaining a version of the tool that would make it possible.

additionally, one would need to call it with `--enable-list`, not with `-t`, as it's an optional rule.

```suggestion
To find out what content is affected after the upgrade, you can run:
[options="nowrap" subs="+quotes,attributes"]
----
# ansible-lint --enable-list only-builtins -p _path/to/your/playbook/play.yml_ ...
----
```